products/redhat-jboss-eap.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /redhat-jboss-eap
+-   /redhatjboss
+-   /redhatjbosseap
+
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes#phases
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
+
+releases:
+-   releaseCycle: "7"
+    support: 2023-06-30
+    eol: 2025-06-30
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
```

Comment:
I do not understand the notion of LTS for JBoss EAP : all versions are LTS and it is not defined in the description. I think this field should be removed.

Reply:
Done :+1: